rskj-core/src/main/resources/reference.conf

```diff
@@ -229,7 +229,7 @@ transaction.gasPriceBump = 40
 transaction.accountSlots = 16
 
 # flag to enable rate-limit for accounts broadcasting transactions consuming large amounts of resources
-transaction.accountTxRateLimit.enabled = false
+transaction.accountTxRateLimit.enabled = true
```

Comment:
Could you also check other `*.conf` files?

There is an `expected.conf` file that we use as a reference to check what properties and what data type it accepts.